src/sentry/logging/handlers.py

```diff
@@ -67,7 +67,9 @@ def __call__(self, logger, name, event_dict):
 class StructLogHandler(logging.StreamHandler):
     def get_log_kwargs(self, record, logger):
         kwargs = {k: v for k, v in vars(record).items() if k not in throwaways and v is not None}
-        kwargs.update({"level": record.levelno, "event": record.msg})
+        kwargs.update(
+            {"level": record.levelno, "event": record.msg, "severity": record.levelname.upper()}
```

Comment:
actually, could we take off the `.upper()` here? since python specifies uppercase: https://docs.python.org/3.6/library/logging.html#logrecord-attributes